package.json

```
@@ -212,7 +212,7 @@
     "@metamask/stake-sdk": "^1.0.0",
     "@metamask/swappable-obj-proxy": "^2.1.0",
     "@metamask/swaps-controller": "^12.1.0",
-    "@metamask/transaction-controller": "^45.1.0",
+    "@metamask/transaction-controller": "^45.0.0",
```

this will continue to install 45.1.0 if it is available because of the carat. "^45.0.0" will support >= 45.0.0 and < 46.0.0. 

I think we want the specific version here + patches
```suggestion
    "@metamask/transaction-controller": "~45.0.0",
```

---

In the case this happens to not work, we can add specific resolutions e.g.

"@metamask/transaction-controller@npm:^45.1.0": "~45.0.0",